README.md

```diff
@@ -1 +1,24 @@
-# data-engineering-project
\ No newline at end of file
+# data-engineering-project
+
+
+## Branching strategy
+
+DEP-XYZ-Name 
```

Comment:
This could be the text here: "The branching strategy we follow is Gitflow: https://nvie.com/posts/a-successful-git-branching-model/".

I would move the template of the branch name to the section "Branch name convention".